mos-platform/geos-cbm/geos.h

```diff
@@ -0,0 +1,18 @@
+/***********************************************************************
+ * geos.h
+ *
+ * This file contains definitions for use in GEOS applications
+ * with llvm-mos C compiler.
+ *
+ * Transcribed from geoProgrammer manual by DiSc.
+ *
+ ***********************************************************************/
+
+#pragma once
```

Comment:
We prefer specific include guards in the SDK for subtle semantic reasons (mostly to do with the complexity of "flie identity").